path.go

```diff
@@ -28,7 +28,7 @@ type Walker struct {
 
 func GetWalker() Walker {
 	w := Walker{
-		ignoredDirectories: []string{".git", ".direnv", "dist"},
+		ignoredDirectories: []string{".git", ".direnv", "dist", "dependencies"},
```

Comment:
At first I used `vendor`, but the name is preserved in go structure 😢 

dependencies is a second candidate for me, from https://github.com/github-linguist/linguist/blob/916bd8f3df802fa98b0ea85539e67bd0b88ef158/lib/linguist/vendor.yml#L16